integration_test_project/dbt_project.yml

```diff
@@ -30,5 +30,8 @@ models:
 seeds:
   +quote_columns: false
 
+flags:
```

Comment:
Runtime Error
  Do not specify both 'config' in profiles.yml and 'flags' in dbt_project.yml. Using 'config' in profiles.yml is deprecated.
integration_bigquery_1_8_0: exit 2 (1.32 seconds) /home/runner/work/dbt_artifacts/dbt_artifacts/integration_test_project> dbt clean pid=3233

Reply:
Would it make sense to move those out of profiles.yml since it's now deprecated?